src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -2382,16 +2382,38 @@ private bool HasGetChildItemDynamicParameters(ProviderInfo providerInfo)
                         // Since the path contained glob characters or we are recursing and the
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies logging level filtering works
```powershell
Expected $null, but got PSLogItem.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 256
256:         $items | Should -Be $null
```